src/app/qgisapp.cpp

```diff
@@ -5620,20 +5620,20 @@ bool QgisApp::askUserForZipItemLayers( const QString &path )
   }
 
   // if promptLayers=Load all, load all layers without prompting
-  if ( promptLayers == 3 )
+  if ( promptLayers == QgsSublayersDialog::PromptLoadAll )
```

Comment:
using a "switch" statement here would be cleaner

Reply:
Agree!